RELEASE_NOTES.md

```diff
@@ -27,240 +27,185 @@ will not be possible to downgrade from 3.0 to 2.6.x any more.
-  entry at the bottom.
+- A new 'culling' mode has been added added to the lightable view. It
+  displays a fixed number of consecutive images starting from the first
+  selected, and allows you to pan & zoom them. The number of displayed images
+  can be set by the user, and they can be navigated with the mouse wheel
+  and keyboard.
 
-- A quite extensive rewrite of the lighttable has been made to improve
-  drastically the overall performance. This also includes the
-  filmstrip which has also received a big rewrite for the same reasons. The
-  lighttable is now usable on 4K and 5K monitors.
+- A quite extensive rewrite of the lighttable (including the film strip) has
+  been made to greatly improve the overall performance.The lighttable is now
+  usable on 4K and 5K monitors.
 
-- Add new raster mask supports. This is a copy of a parametric mask
+- Added support for the new 'raster mask', a copy of a parametric mask
   which is stable during the whole pixel-pipe.
 
-- Modules can now be re-ordered with <kbd>Ctrl+Shift+drag</kbd>.
-
-  IMPORTANT PLEASE READ:
-
-  Note that this is NOT a GUI re-ordering but a pixel-pipe
-  reordering. If you're not familiar with pixel pipe order and the
-  actual quality that could result from changing the default, do not
-  change the order.
-
-  The default order has been set to give the best result in the
-  majority of time and will never be on the wrong side.
-
-- The color zones module now display an histogram based on the ‘select
-  by’ channel and displays the selected range if the color picker is
-  in area mode.
-  It acts now like the tone curve (but horizontal),
-  it has two edit modes: edit by area is the former one, if not checked nodes
-  can be edited like in the tone curve, delete only works by right-click
-  when not in edit area mode.
-  A new process mode has been added: smooth is the former one, strong is new.
-  An interpolation method has been added that allows to select different types
-  of curves.
-  The color picker allows to select by area with <kbd>Ctrl+click</kbd>.
-  A new color picker has been added that creates a curve based on the area
-  selected from the image.
-  When pressing the shortcut 'pan&zoom while editing masks' the draw area
-  can now be zoomed. Double-click on the bottom bar resets the zoom.
-
-- New module Filmic RBG. This module is designed to replace the base
-  curve, shadows and highlights. The new filmic rgb module replaces
-  the old one introduced in 2.6.2 and should be easier to use and will
-  reduce color casts. The old filmic module is now deprecated and
-  cannot be used on new developments.
-
-- New module tone equalizer which brings an easy and safe way to
-  adjust the tones in different areas of the image. The module comes
-  with 9 controls ranging from blacks area to specular ones. It
-  possible to adjust each area with slides, curve or directly over the
-  image with a controllable picker.
```

Comment:
Information that edit can be done using sliders, curves or directly o the image is kind of lost in the edit.

Reply:
Same response as color zones above.  :-)

I may have done too much condensing, but these are release notes, not the manual, and I'm a little bit afraid some people will run out of attention span.

Reply:
Fair enough.